requirements.txt

```diff
@@ -1,4 +1,8 @@
-setuptools~=49.6.0
-numpy~=1.20.3
-mdanalysis~=2.0.0.dev0
-pytest~=6.2.4
\ No newline at end of file
+setuptools
+numpy>=1.16.0
+mdanalysis>=2.0.0
```

Comment:
Your test environment is still pinned to 1.1.1, setting to 2.0.0 here doesn't seem necessary?